### DISCUSSION TITLE
Deploying Durable Functions to Azure Container Apps

### DISCUSSION THREAD

**Discussion by shibayan** [Azure/azure-functions-durable-extension#2159] [Show and tell]
I have created a Bicep template to deploy Durable Functions on Azure Container Apps and share it with you.

This example uses Azure Storage Provider as the backend, and scaling is generally working as expected.

## Bicep template

https://gist.github.com/shibayan/21290078e52d3de187fd820491a24f03

## Created resources

![image](https://user-images.githubusercontent.com/1356444/166423121-10efa91f-c14b-4269-97fb-d6cf7e3d0c83.png)

## Appendix : MSSQL Provider (for scale to zero)

Durable Functions developer @cgillum says that for scaling to zero, it is better to use MSSQL Provider.

https://twitter.com/cgillum/status/1521184880704466944

**Comment by davidmrdavid** [comment]
Oh very cool, thank you for sharing :)
